src/main/kotlin/com/openlattice/organizations/HazelcastOrganizationService.kt

```diff
@@ -523,34 +539,37 @@ class HazelcastOrganizationService(
     }
 
     fun removeUser(principal: Principal) {
-        organizations.executeOnEntries(OrganizationEntryProcessor { it.members.remove(principal) })
+        organizations.executeOnEntries(OrganizationEntryProcessor {
+            val modified = it.members.remove(principal)
+            Result(modified, modified)
+        })
     }
 
     fun updateRoleGrant(organizationId: UUID, roleId: UUID, grant: Grant) {
         organizations.executeOnKey(organizationId, OrganizationEntryProcessor {
             it.grants.getOrPut(roleId) { mutableMapOf() }[grant.grantType] = grant
-            null
+            Result(null) //TODO: Being lazy not implementing diff as this should rarely be called.
```

Comment:
Looks like the build is failing without handling this case tho

```OrganizationEntryProcessorStreamSerializerTest.kt: (39, 13): Null can not be a value of a non-null type OrganizationEntryProcessor.Result```

Reply:
The entry processor in the test wasn't wrapping the result. Should be fixed now.